src/main/java/com/javadiscord/javabot/events/AutoMod.java

```diff
@@ -55,6 +56,10 @@ public void onGuildMessageReceived(@NotNull GuildMessageReceivedEvent event) {
             }
         });
 
+        // known phishing link
+        if (event.getMessage().getContentRaw().contains("https://discordc.gift/")){
+            ban(event, member, "scam");
```

Comment:
I don't think they should. People are stupid and they will post links on accident or because they want to know whether it's harmful or not and just don't know the damages it might cause to others.